external/storm-druid/src/main/java/org/apache/storm/druid/bolt/DruidBeamBolt.java

```diff
@@ -78,27 +76,28 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
 
     @Override
     protected void process(final Tuple tuple) {
-        Future future = tranquilizer.send((druidEventMapper.getEvent(tuple)));
-        LOG.debug("Sent tuple : [{}]", tuple);
+        final E mappedEvent = druidEventMapper.getEvent(tuple);
+        Future future = tranquilizer.send(mappedEvent);
+        LOG.debug("Sent tuple : [{}]", mappedEvent);
 
         future.addEventListener(new FutureEventListener() {
             @Override
             public void onFailure(Throwable cause) {
                 if (cause instanceof MessageDroppedException) {
                     collector.ack(tuple);
-                    LOG.debug("Tuple Dropped due to MessageDroppedException : [{}]", tuple);
+                    LOG.debug("Tuple Dropped due to MessageDroppedException : [{}]", mappedEvent, cause);
```

Comment:
nit: I am assuming that you are adding the cause here because you want to debug it more.  Do you really need the full stack trace?  I don't see that changing between different times it happened.  Could you instead set it to something like the following?

```
LOG.debug("Tuple Dropped due to MessageDroppedException {} : [{}]", cause.getMessage(), mappedEvent);
```

Reply:
exception message should be sufficeint. Updated the PR.